### PR TITLE
Update profiling docs

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Update profiling docs

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/8084, https://github.com/pingcap/docs/pull/7417

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ngaut — APPROVED** [review]
LGTM

**Review by breeswish — APPROVED** [review]
Thanks!

File: dashboard/dashboard-faq.md [hunk 1]
```diff
@@ -74,9 +74,9 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 
     ![清理 Local Storage](/media/dashboard/dashboard-faq-devtools-application.png)
 
-### 界面提示 NgMonitoring 组件未能正常启用
+### 界面提示 `集群中未启动必要组件 NgMonitoring`
 
-如果 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 页面提示 NgMonitoring 组件未能正常启用，你可以通过以下步骤逐步排查。 
+部署 TiDB 集群时，NgMonitoring 会自部署，无需手动操作。如果 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 页面提示 NgMonitoring 组件未能正常启用，你可以通过以下步骤逐步排查。
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:79]: 在 5.4.0 版本中有两个功能都会依赖于它，这样如何？

```
NgMonitoring 是 TiDB v5.4.0 及以上集群中内置的高级监控组件，用于支撑 TiDB Dashboard 的 **持续分析** (Continuous Profiling)、**Top SQL** 等功能。使用 TiUP 部署集群时，NgMonitoring 会自动部署；使用 TiDB Operator 部署集群时，需要依据升级及部署文档手工部署。

若已使用 TiUP 进行部署，访问 TiDB Dashboard 页面后出现 NgMonitoring 组件未启动相关提示，你可以通过以下步骤逐步排查。

……

若你使用 TiDB Operator 进行部署，请参见 [TiDB Operator 相关文档](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/access-dashboard/#启用持续性能分析)。
```

File: dashboard/continuous-profiling.md [hunk 2]
```diff
@@ -30,6 +30,8 @@ summary: TiDB Dashboard 持续性能分析功能 (Continuous Profiling)
 
 ## 启用持续性能分析
 
+本节介绍如何在使用 TiUP 和 TiDB Operator 部署的 TiDB 集群上启用持续性能分析。
```
Comment (qiancai) [inline dashboard/continuous-profiling.md:33]:
```suggestion
本节介绍如何在使用 TiUP 或 TiDB Operator 部署的 TiDB 集群上启用持续性能分析。
```

File: dashboard/dashboard-faq.md [hunk 3]
```diff
@@ -74,41 +74,39 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 
     ![清理 Local Storage](/media/dashboard/dashboard-faq-devtools-application.png)
 
-### 界面提示 NgMonitoring 组件未能正常启用
+### 界面提示 `集群中未启动必要组件 NgMonitoring`
 
-如果 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 页面提示 NgMonitoring 组件未能正常启用，你可以通过以下步骤逐步排查。 
+NgMonitoring 是 TiDB v5.4.0 及以上集群中内置的高级监控组件，用于支撑 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 和 **Top SQL** 等功能。使用 TiUP 部署集群时，NgMonitoring 会自动部署；使用 TiDB Operator 部署集群时，需要依据[启用持续性能分析](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/access-dashboard/#%E5%90%AF%E7%94%A8%E6%8C%81%E7%BB%AD%E6%80%A7%E8%83%BD%E5%88%86%E6%9E%90)手工部署 NgMonitoring。
```
Comment (qiancai) [inline dashboard/dashboard-faq.md:79]: @breeswish NgMonitoring 是 TiDB v5.4.0 及以上集群中内置的高级监控组件? 这里的  5.4.0 要改成 5.3.0 吗？

Comment (breeswish) [reply dashboard/dashboard-faq.md:79]: 5.3.0 不会默认部署，仅 5.4.0 会默认部署

File: dashboard/dashboard-faq.md [hunk 4]
```diff
@@ -118,7 +116,7 @@ NgMonitoring 组件需要较高版本的部署工具支持（TiUP 1.9.0 及以
 tiup cluster reload ${cluster-name} --role prometheus
 ```
 
-#### 第 3 步：配置 TiDB Dashboard
+第 3 步：配置 TiDB Dashboard
```
Comment (breeswish) [inline dashboard/dashboard-faq.md:119]: 作为错误提示的应对的话，L119 ~ L123 这里的「第三部」是可以去掉的，做完前两步后，应该不会再提示 ngm 错误了。

**Review by breeswish — APPROVED** [review]
Looks good, thanks!

File: dashboard/dashboard-faq.md [hunk 5]
```diff
@@ -74,41 +74,41 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
 
     ![清理 Local Storage](/media/dashboard/dashboard-faq-devtools-application.png)
 
-### 界面提示 NgMonitoring 组件未能正常启用
+### 界面提示 `集群中未启动必要组件 NgMonitoring`
 
-如果 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 页面提示 NgMonitoring 组件未能正常启用，你可以通过以下步骤逐步排查。 
+NgMonitoring 是 TiDB v5.4.0 及以上集群中内置的高级监控组件，用于支撑 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 和 **Top SQL** 等功能。使用 TiUP 部署集群时，NgMonitoring 会自动部署；使用 TiDB Operator 部署集群时，需要依据[启用持续性能分析](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/access-dashboard/#%E5%90%AF%E7%94%A8%E6%8C%81%E7%BB%AD%E6%80%A7%E8%83%BD%E5%88%86%E6%9E%90)手工部署 NgMonitoring。
```
Comment (qiancai) [inline dashboard/dashboard-faq.md:79]:
```suggestion
NgMonitoring 是 TiDB v5.4.0 及以上集群中内置的高级监控组件，用于支撑 TiDB Dashboard 的 **持续分析** (Continuous Profiling) 和 **Top SQL** 等功能。使用 TiUP 部署集群时，NgMonitoring 会自动部署；使用 TiDB Operator 部署集群时，需要依据[启用持续性能分析](https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/access-dashboard/#%E5%90%AF%E7%94%A8%E6%8C%81%E7%BB%AD%E6%80%A7%E8%83%BD%E5%88%86%E6%9E%90)手动部署 NgMonitoring。
```

File: dashboard/dashboard-faq.md [hunk 6]
```diff
@@ -74,41 +74,41 @@ QPS 及 Latency 监控依赖于集群中已正常部署 Prometheus 监控实例
...
+如果界面提示 `集群中未启动必要组件 NgMonitoring`，需要按 TiDB 集群部署方式排查问题。
 
-NgMonitoring 组件需要较高版本的部署工具支持（TiUP 1.9.0 及以上）。检查 TiUP Cluster 版本，若版本低于 1.9.0，则需要先升级 TiUP Cluster。
+#### 使用 TiUP 部署的集群
 
-1. 检查 TiUP Cluster 版本：
+第 1 步：检查 TiUP Cluster 版本
 
-    {{< copyable "shell-regular" >}}
+  1. 检查 TiUP Cluster 版本，NgMonitoring 组件需要较高版本的部署工具支持（TiUP 1.9.0 及以上）：
```
Comment (qiancai) [inline dashboard/dashboard-faq.md:87]:
```suggestion
  1. 检查 TiUP Cluster 版本，NgMonitoring 组件需要较高版本的部署工具支持（TiUP v1.9.0 及以上）：
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**